backtracking/all-combinations-of-size-k.ts

```diff
@@ -0,0 +1,25 @@
+export function generateCombinations(n: number, k: number): number[][] {
+  return generateAllCombos(n, k);
+}
+
+function generateAllCombos(
+  n: number,
+  k: number,
+  startCursor: number = 1,
```

Comment:
I wouldn't make these default arguments. Rather, I'd just set them initially in the call on line 2.